test/ee2_scheduler_kafka_test.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+"""Module to test out logging to kafka."""
+
+import json
+import logging
+
+from execution_engine2.utils.KafkaUtils import (
+    KafkaHandler,
+    send_message_to_kafka,
+    _test_sample_consumer,
+)
+import unittest
+
+
+class ExecutionEngine2SchedulerTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        """
+        This test is used for sending commands to a live environment, such as CI.
+        TravisCI doesn't need to run this test.
+        :return:
+        """
+
+    @classmethod
+    def tearDownClass(cls):
+        pass
+
+    def test_kafka_producer(self):
+        message = {
+            "heading": "Dear Dolly",
+            "body": "Thank you for being a great cloned sheep.",
+        }
+        send_message_to_kafka(data=message)
```

Comment:
Can you add assertions here? One way to do it is to produce, then run a test consumer that consumes until you get the message you are expecting, with a max timeout of something like 180seconds